packages/core/src/interaction-output.ts

```diff
@@ -70,6 +73,10 @@ export class InteractionOutput implements WoT.InteractionOutput {
             throw new NotSupportedError(message);
         }
 
+        if (this._stream) {
+            throw new Error("Can't call value function after retrieving the stream");
+        }
```

Comment:
I think this error needs to be added to both functions

`arrayBuffer() ` AND `value<T>()`